src/templates/includes/country.html

```diff
@@ -0,0 +1,250 @@
+<select name="country">
+  <option value="">Country</option>
+  <option value="Afganistan">Afghanistan</option>
```

Comment:
it looked like on the site u pulled these from they had 2 options, one for full name as value and another abbreviated one. might be best to use abbreviated as it makes for cleaner data? probably a preference thing tho. full name is good for the actual text in the select tho.